locales/de/translation.json

```diff
@@ -238,5 +238,11 @@
   "Plug in meter with cable and set meter to": "Messgerät mit Kabel anschließen und Messgerät auf ",
   "PC Link Mode": "PC-Link-Modus",
   "afterLink": " einstellen",
-  "Make sure the meter is switched off and plug in with micro-USB cable": "Stellen Sie sicher, dass das Messgerät ausgeschaltet ist und schließen Sie es mit einem Micro-USB-Kabel an"
+  "Make sure the meter is switched off and plug in with micro-USB cable": "Stellen Sie sicher, dass das Messgerät ausgeschaltet ist und schließen Sie es mit einem Micro-USB-Kabel an",
+  "e-instructions": "e-instructions",
```

Comment:
Tässä "oikea" avain olisi `eIFU indicator`. Suomeksi `sähköisen käyttöohjeen sijainti`. Ja sen olisi hyvä olla vähintäänkin title-attribuuttina, ehkä myös ihan kokonaan näkyvissä.

Tästä kertoo tarkemmin ISO-standardi 15223, joka meillä on käytössämme. @HannuHyttinen kertoo tarvittaessa lisää.

Reply:
Ja tämä siis kaikilla kielillä. :)

Reply:
Ja se title-attribuutin sisältö tai symbolin vieressä näkyvä teksti olisi ehkä standardin mukaan "Consult instructions for use" / "Katso käyttöohjeet, ohjeet sähköisessä muodossa". Vähän vaikeasti tulkittava tuo ohjestus ehkä on...

Reply:
Lisäsin tuon avaimen sekä title-attribuutiksi, sekä kokonaan näkyville, mutta myöhemmin poistin title-attribuutista, sillä sama teksti oli myös näkyvissä siinä vieressä.